docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+services:
+  webpack:
+    container_name: dim-webpack
+    build:
+      context: ./docker-compose/webpack
+    command: bash -c "yarn install && yarn start"
+    restart: always
+    volumes:
+      - .:/usr/src/app
+  http-server:
+    container_name: dim-http-server
+    build:
+      context: ./docker-compose/http-server
+    ports:
+      - "8080:8080"
+    restart: always
+    volumes:
+      - .:/usr/src/app
```

Comment:
Can you create named volumes instead of using host-mounted volumes? It's going to make the setup more compatible with automated systems, as well as users who might use orchestration platforms as their development bases.

Reply:
Good stuff, working on this now

Reply:
@LongLiveCHIEF 

Your thoughts on this are more then welcome, it sounds like you might have a bit more experience in this area then me

I'm not totally sure about this, the host mounted volume is to give the developer using this the ability to edit files on their local computer without having to rebuild the image or starting an ssh client in the image which is not really appropriate for docker

I thought about moving the volumes config for each image into the volumes directive to indicate more accurately that both images are using the same files, but I am not familiar with a way to get that to work as a relative path. If we do not use relative path, then everybody would have to edit a file to put the correct path depending on their host OS, which makes the setup more complicated then the single command it is now.

I think maybe if this application was deployed with an orchestration tool we would have a totally different setup that would just generate an image with the dist files and http-server for the production build

The intention of this was to make it really fast and easy for people to start development on any platform without having to setup any of the node services